js/skip-nav.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/* global require, module, document */
+
+var $ = require('jquery');
+
+function Skipnav(selector) {
+  var self = this;
```

Comment:
Probably no need to alias `self` to `this` here, since we aren't using any closures in this method.